server/server.go

```diff
@@ -31,17 +31,20 @@ import (
 
 // Server is the mattermod server.
 type Server struct {
-	Config           *Config
-	Store            store.Store
-	GithubClient     *GithubClient
-	CircleCiClient   CircleCIService
-	CircleCiClientV2 CircleCIService
-	OrgMembers       []string
-	Builds           buildsInterface
-	commentLock      sync.Mutex
-	StartTime        time.Time
-	awsSession       *session.Session
-	Metrics          MetricsProvider
+	Config                *Config
+	Store                 store.Store
+	GithubClient          *GithubClient
+	CircleCiClient        CircleCIService
+	CircleCiClientV2      CircleCIService
+	OrgMembers            []string
+	Builds                buildsInterface
+	commentLock           sync.Mutex
+	StartTime             time.Time
+	awsSession            *session.Session
+	Metrics               MetricsProvider
+	cherryPickRequests    chan *cherryPickRequest
+	cherryPickStopChan    chan struct{}
+	cherryPickStoppedChan chan struct{}
```

Comment:
-100/5 nit, since the other channels have a "chan" suffix, then I think `cherryPickRequests` should be `cherryPickRequestsChan` too

Reply:
I think `cherryPickRequests` fits nice on the purpose, but the other cases `stop` and `stopped` seem to be quite dull without a `chan` suffix.